docs/getting-started.rst

```diff
@@ -17,10 +16,226 @@ Creating a New Project
+
+        pub_date = serializers.DateTimeField(read_only=True)
+        content =  serializers.CharField()
+        user = serializers.PrimaryKeyRelatedField(
+            queryset=User.objects.all(),
+            pk_field=serializers.UUIDField(format="hex_verbose"),
+        )
+        post = serializers.PrimaryKeyRelatedField(
+            queryset=Post.objects.all(),
+            pk_field=serializers.UUIDField(format="hex_verbose"),
+        )
+
+        class Meta:
+            model = Comment
+            proto_class = CommentResponse
+            proto_class_list = CommentListResponse
+            fields = "__all__"
+
+
+Defining gRPC services
+~~~~~~~~~~~~~~~~~~~~~~~
+Django Socio gRPC uses the name Service instead of View or Viewset.
+With the exception of the name and the internal layer, a gRPC service works in the same way as a generic DRF View.
+
+Django Socio gRPC Framework actually support both async and sync mode for gRPC.
+
+You can refer to the part of the documentation describing the two types of method. In this example we are using an asynchronous service.
+
+Following the same logic as DRF, Django Socio gRPC uses class-based services.
+
+Here we implement pagination, permissions and filters by way of example.
+You can write a mixin including these parameters and make an inheritance on the service class.
+Please refer to the :ref:`Mixin section <Generic Mixins>` of this documentation to do this.
+
+  .. code-block:: python
+
+    #quickstart/services.py
+    from django_filters.rest_framework import DjangoFilterBackend
+    from rest_framework.pagination import PageNumberPagination
+    from rest_framework.permissions import BasePermission
+    from django_socio_grpc import generics
+
+    from quickstart.models import User, Post, Comment
+    from quickstart.serializer import UserProtoSerializer, PostProtoSerializer, CommentProtoSerializer
+
+
+    class UserService(generics.AsyncBaseService):
+
+        pagination_class = PageNumberPagination
+        permission_classes = (BasePermission,)
+        filter_backends = [DjangoFilterBackend]
+
+        queryset = User.objects.all()
+        serializer_class = UserProtoSerializer
+    
+    class PostService(generics.AsyncBaseService):
+        queryset = Post.objects.all()
+        serializer_class = PostProtoSerializer
+    
+    class CommentService(generics.AsyncBaseService):
+        queryset = Comment.objects.all()
+        serializer_class = CommentProtoSerializer
+
+**Note:**
+
+You have the flexibility to revert to using the classic APIVIEW class or to reuse mixins or base classes to adapt to your needs. 
+
+Example:
+
+  .. code-block:: python
+
+    from django.contrib.auth.models import User
+    from quickstart.serializers import UserProtoSerializer
+    from django_socio_grpc import generics
+
+    class UserListService(generics.ListCreateAPIView):
+            queryset = User.objects.all()
+            serializer_class = UserProtoSerializer
```

Comment:
This is not true there's no APIVIEW in DSG

Reply:
I copied this part of the current documentation https://socotecio.github.io/django-socio-grpc/generic_service/ . I thought it would be interesting to suggest. However, there was an error in the import of the generics, it was from rest_framework, not DSG. Should we keep this part? In the meantime I've corrected the import error.

Reply:
Done